BiasAndFairnessModule/run_full_evaluation.py

```
@@ -0,0 +1,345 @@
+    priority_metrics = user_selected_metrics + [m for m in fairness_compass_recommended_metrics if m not in user_selected_metrics]
+    print(f"\n🎯 Priority Order for Evaluation:")
+    print(f"   1. User Selected: {user_selected_metrics}")
+    print(f"   2. Fairness Compass Additional: {[m for m in fairness_compass_recommended_metrics if m not in user_selected_metrics]}")
+    print(f"   3. Other Available: {[m for m in all_available_metrics if m not in priority_metrics]}")
+    
+    # Run metrics in priority order
+    print(f"\n🔍 Running Fairness Metrics in Priority Order...")
+    all_results = {}
+    
+    # 1. HIGHEST PRIORITY: User selected metrics from config.yaml
+    print(f"\n🎯 1. User Selected Metrics (config.yaml) - HIGHEST PRIORITY:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)